mirgecom/flux.py

```diff
@@ -35,6 +44,40 @@
 import numpy as np  # noqa
 
 
+def num_flux_lfr(f_minus, f_plus, q_minus, q_plus, lam, **kwargs):
+    """Lax-Friedrichs/Rusanov low-level numerical flux."""
+    return (f_minus + f_plus - lam*(q_plus - q_minus))/2
+
+
+def num_flux_central(f_minus, f_plus, **kwargs):
+    """Central low-level numerical flux."""
+    return (f_plus + f_minus)/2
+
+
+def hll_flux_driver(state_pair, physical_flux_func,
+                    s_minus, s_plus, normal):
+    """State-to-flux driver for hll numerical fluxes."""
+    actx = state_pair.int.array_context
+    zeros = 0.*state_pair.int.mass_density
+
+    f_minus = physical_flux_func(state_pair.int)@normal
+    f_plus = physical_flux_func(state_pair.ext)@normal
+    q_minus = state_pair.int.cv
+    q_plus = state_pair.ext.cv
```

Comment:
If this is intended to be generic across conservation laws, it should not access `.cv`.

Reply:
Fixed, I think.

Reply:
#648